assets/schema/ensemble_schema.json

```diff
@@ -956,6 +956,52 @@
         }
       }
     },
+    "ListView-payload": {
+      "type": "object",
+      "allOf": [
+        {
+          "properties": {
+            "children": { "$ref": "#/$defs/Widgets" },
+            "onItemTap": {
+              "description": "Dispatch when an ListView item is selected/tapped.The event dispatches only when you tap on the item. The index of the item can be retrieved using 'selectedItemIndex'.",
+              "$ref": "#/$defs/Action-payload"
+            },
+            "styles": {
+              "allOf": [
+                {
+                  "$ref": "#/$defs/boxLayoutStyles"
+                },
+                {
+                  "$ref": "#/$defs/baseStyles"
+                },
+                {
+                  "type": "object",
+                  "properties": {
+                    "separatorColor": {
+                      "$ref": "#/$defs/typeColors",
+                      "description": "Set the color for the separator between items"
+                    },
+                    "separatorWidth": {
+                      "type": "integer",
+                      "description": "The thickness of the separator between items"
+                    },
+                    "separatorPadding": {
+                      "type": ["integer", "string"],
+                      "description": "Padding with CSS-style value e.g. padding: 5 20 5 Default 0 0 0"
+                    }
+                  }
+                }
+              ]
+            },
+            "selectedItemIndex": {
```

Comment:
move this to before styles. The either is to list properties together, then styles afterward. It's a huge file and easier to debug/review.